packages/gafl-webapp-service/src/pages/summary/contact-summary/contact-summary.njk

```diff
@@ -6,7 +6,7 @@
 {% from "back-link/macro.njk" import govukBackLink %}
 
 {% set title = 'Check the contact details' %}
-{% block pageTitle %}{{ title }} - GOV.UK{% endblock %}
+{% block pageTitle %}{{ title }}{{ mssgs.gov_uk }}{% endblock %}
 
 {% set licenseeDetailsSummary %}
     {% call govukFieldset({
```

Comment:
The rows on the contact summary haven't been translated. This will be slightly trickier as the text is in the route.js.